### PR TITLE
Newlines / ordering of using statements / consistency

### PR DESCRIPTION
**Pull request by fretje**
Unnecessary newlines have been removed.
A newline is added at the end of files where there was none.
Using statements have been ordered and removed where not needed.
StringLength is now used everywhere (iso MaxLength).
Spaces have been added to comments where necessary.
Capitalization of comments.

Consistency in general... 

Only done now for the server part... might do the client part later.

Let me know if this is ok... 
I thought of also using expression body for methods where possible... but that might be a bridge too far? ;-)

### CODE REVIEW

**Comment by fretje** [comment]
Btw, I think it would also be more consistent to use `CreateTenantInput`, `CreateUserInput` and `CreateRoleInput` in stead of `CreateTenantDto`, `CreateUserDto` and `CreateRoleDto`.

That would be for a separate PR anyway, as it would also need changes on the client side... but what would be your opinion on this?

File: aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/Dto/TenantDto.cs [hunk 1]
```diff
@@ -5,18 +5,18 @@
 
 namespace AbpCompanyName.AbpProjectName.MultiTenancy.Dto
 {
-    [AutoMapTo(typeof(Tenant)), AutoMapFrom(typeof(Tenant))]
+    [AutoMapFrom(typeof(Tenant))]
```
Comment (acjh) [inline aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/Dto/TenantDto.cs]: `[AutoMap(typeof(Tenant))]`

Comment (fretje) [reply aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/Dto/TenantDto.cs]: Actually, the AutoMapTo is not used, as that mapping is done manually in TenantAppService (see https://github.com/aspnetboilerplate/module-zero-core-template/blob/master/aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/TenantAppService.cs#L98)

**Comment by hikalkan** [comment]
Thank you for your huge contributon, it was long to review :)

**Pull request merged**